### PR TITLE
feat(dashboard): add dashboard for core node components health

### PR DESCRIPTION
**Pull request by Atrax1**
Core Node components dashboard

### CODE REVIEW

File: dashboard-lib/core-node-components/component.go [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+			query.IncludeAll(),
+			query.Request(`label_values(up{env="$env", cluster="$cluster", blockchain="$blockchain", network_type="$network_type", component="$component"}, service)`),
+			query.Sort(query.NumericalAsc),
+		),
+		dashboard.VariableAsQuery(
+			"service_id",
+			query.DataSource(p.PrometheusDataSource),
+			query.Multiple(),
+			query.IncludeAll(),
+			query.Request(`label_values(health{env="$env", cluster="$cluster", blockchain="$blockchain", network_type="$network_type", component="$component", service="$service"}, service_id)`),
```
Comment (patrickhuie19) [inline dashboard-lib/core-node-components/component.go:81]: This approach makes sense to me - the queries drill down by each level of granularity. Is there value in allowing some of them (i.e. product or service) to not necessarily need a higher level up if I want to quickly compare across envs? Or does grafana give us that for free?

File: dashboard-lib/core-node-components/platform.go [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+// PlatformPanelOpts generate different queries for "docker" and "k8s" deployment platforms
+func PlatformPanelOpts() PlatformOpts {
+	po := PlatformOpts{
+		LabelFilters: map[string]string{
+			"env":          `=~"${env}"`,
+			"cluster":      `=~"${cluster}"`,
+			"blockchain":   `=~"${blockchain}"`,
+			"product":      `=~"${product}"`,
+			"network_type": `=~"${network_type}"`,
```
Comment (patrickhuie19) [inline dashboard-lib/core-node-components/platform.go:20]: I can't access grafana through the vpn currently, so sorry for the trivial question - what values does network_type take on?

File: dashboard-lib/core-node-components/component.go (hunk 1 above)
Comment (Atrax1) [reply dashboard-lib/core-node-components/component.go:81]: You don't have to specify all of them look at this example
image

File: dashboard-lib/core-node-components/platform.go (hunk 2 above)
Comment (Atrax1) [reply dashboard-lib/core-node-components/platform.go:20]: mainnet or testnet

**Review by patrickhuie19 — APPROVED** [review]

**Pull request merged**